src/FacebookPosterPost.php

```diff
@@ -105,6 +106,10 @@ public function withVideo($videoPath, $data = [], $endpoint = 'me/videos')
      */
     public function scheduledFor($timestamp)
     {
+        $timestamp = $timestamp instanceof DateTime
```

Comment:
Great idea! But i'd use `DateTimeInterface`, which means it will also accept immutable variants (`DatetimeImmutable` & `CarbonImmutable`)

Reply:
Good call, updated.